### PR TITLE
Bump dependencies 

### PR DESCRIPTION
**Pull request by johnatasr**
#### Proposed Changes ####

- Bump for Charts
-  Dockerfile images
- build script images

#### User-Facing Change ####

```release-note

```

### CODE REVIEW

**Review by brandond — APPROVED** [review]

**Review by dereknola — APPROVED** [review]

File: scripts/build-windows-binary
Comment (brandond) [inline scripts/build-windows-binary:33]: Also need to change this in https://github.com/rancher/rke2/blob/45c21222401f736f35c0ceff66db695c195b89a5/scripts/build-binary#L27

**Review by brandond — CHANGES_REQUESTED** [review]

**Review by briandowns — APPROVED** [review]

**Review by brandond — APPROVED** [review]
LGTM, please squash before or when merging.

**Comment by manuelbuil** [comment]
@johnatasr Please note that changes must be backported to previous releases. As far as I see, this PR was never backported to 1.27 and 1.26 branches. Note as well that PRs must always be linked to issues so that QA can verify them

**Pull request merged**
